tests/data/repos/conda_exercises_fleeqtk/fleeqtk_seq.xml

```diff
@@ -0,0 +1,102 @@
+<tool id="fleeqtk_seq" name="Convert to FASTA (fleeqtk)" version="0.1.0">
+    <stdio>
```

Comment:
Yeah - but this was based on the seqtk tutorial and tool_init. I have created an issue for this here #644. I'll get to this before say the GCC I think - but I want to do every variant of seqtk in the code base at once after that issue is addressed.